test/unittests/parser_test.cpp

```diff
@@ -63,7 +63,7 @@ TEST(parser, DISABLED_limits_min_invalid_too_short)
     EXPECT_THROW(parser<Limits>{}(input.data()), parser_error);
 }
 
-TEST(parser, DISABLED_limits_minmax_invalid_too_short)
+TEST(parser, limits_minmax_invalid_too_short)
```

Comment:
How is this detected now?

Reply:
No idea.

Reply:
Is ASan running properly? Shouldn't that detect overreads?

Reply:
Now it does.